challenges/functions.js

```diff
@@ -1,32 +1,49 @@
-// ==== Callbacks ====  
+// ==== Callbacks ====
 
 /* Step 1: Create a higher-order function that accepts a callback
-  * Create a higher-order function named consume that can take 3 parameters.
-  * The first two parameters can accept any argument
-  * The last parameter accepts a callback 
-  * In the body of the function return the callback with the two parameters that you created
-*/
-
+ * Create a higher-order function named consume that can take 3 parameters.
+ * The first two parameters can accept any argument
+ * The last parameter accepts a callback
+ * In the body of the function return the callback with the two parameters that you created
+ */
+
+function consume(x, y, cb) {
+  return cb(x, y);
```

Comment:
We already talked about this, but you can avoid having to write a return and console log in all the the callback functions by console logging the consume function.